### ISSUE TITLE
Instance group creation timeout

### ISSUE THREAD

**Issue by un1x01d** [GoogleCloudPlatform/terraform-google-nat-gateway#71]
Hi, 
Trying to create a simple nat setup using this simple config: 

```
module "nat" {
  source     = "GoogleCloudPlatform/nat-gateway/google"
  region     = "us-east1"
  network    = "tfstate-vpc-nat"
  subnetwork = "tfstate-vpc-nat-subnet"
}

module "mig" {
  source      = "github.com/GoogleCloudPlatform/terraform-google-managed-instance-group"
  region      = "us-east1"
  zone        = "us-east1-b"
  name        = "nat-group"
  target_tags = ["${module.nat.routing_tag_regional}"]
  network    = "tfstate-vpc-nat"
  subnetwork = "tfstate-vpc-nat-subnet"
  service_port      = 80
  service_port_name = "http"
}
```
When creating the instance group, i can see it in the GCP console but it shows that it's still creating until it times-out after 20mins.

```
module.nat.module.nat-gateway.google_compute_instance_group_manager.default: Creating...
  auto_healing_policies.#:                   "" => "1"
  auto_healing_policies.0.health_check:      "" => "https://www.googleapis.com/compute/v1/projects/tfstate/global/healthChecks/nat-gateway-us-east1-b"
  auto_healing_policies.0.initial_delay_sec: "" => "300"
  base_instance_name:                        "" => "nat-gateway-us-east1-b"
  description:                               "" => "compute VM Instance Group"
  fingerprint:                               "" => "<computed>"
  instance_group:                            "" => "<computed>"
  instance_template:                         "" => "https://www.googleapis.com/compute/beta/projects/tfstate/global/instanceTemplates/default-20180725205407178200000001"
  name:                                      "" => "nat-gateway-us-east1-b"
  named_port.#:                              "" => "1"
  named_port.0.name:                         "" => "http"
  named_port.0.port:                         "" => "80"
  project:                                   "" => "<computed>"
  self_link:                                 "" => "<computed>"
  target_size:                               "" => "1"
  update_strategy:                           "" => "NONE"
  version.#:                                 "" => "<computed>"
  wait_for_instances:                        "" => "true"
  zone:                                      "" => "us-east1-b"
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (10s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (20s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (30s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (40s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (50s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m0s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m10s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m20s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m30s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m40s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (1m50s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (2m0s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (2m10s elapsed)
module.nat.nat-gateway.google_compute_instance_group_manager.default: Still creating... (2m20s elapsed)

..... .... ...
```

**Comment by JessicaGreben** [comment]
I was having the same problem yesterday where `module.nat.nat-gateway.google_compute_instance_group_manager.default` was timing out.  In the GCP console, the Instance Group had an infinite spinner saying it was still trying to "Verify".  Magically it seems to have resolved itself today.  Wish I had more details why, maybe something on GCP's side. /shrug.

**Comment by danisla** [comment]
Creating the health check for the self-healing of instances in the managed instance group can take several minutes to configure on the GCP network side. This should work most if not all of the time and I expect it to resolve itself over time.

**Comment by danisla** [comment]
Fixed with #79

**Comment by Logan-Kang** [comment]
Hello

As above, I also have timeout issue is also occurring when using managed instance group module.
If the problem is not generated at once, check instance_group_manager for more than 5 minutes during regeneration attempts, and this also does not run, so destroy is also impossible. Please tell me how to solve this problem.
